src/utils/testing/mocks/mockTag.ts

```diff
@@ -0,0 +1,22 @@
+import { mockObject } from 'utils/testing/mocks';
+import { ZetkinTag } from 'types/zetkin';
+
+const tag: ZetkinTag = {
+  color: null,
+  description: 'People who organise',
+  hidden: false,
+  id: 1,
+  organization: {
+    id: 1,
+    title: 'KPD',
+  },
+  title: 'Organiser',
```

Comment:
Should we care about s/z (British/American) in strings? I don't, really, care too much, the way I do when it comes to actual symbols in the code (e.g. variables).

Reply:
I think in tests it doesn't matter really

Reply:
I changed this so it's consistent everywhere. Using USA spellings :us: